### PR TITLE
Further additions to changelog

### PR DESCRIPTION
**Pull request by aubergine10**
Will be adding TM+AI notes to changelog via this PR to complete entire history of TM:PE and all its predecessors.

Added missing issue ref in 10.17 notes

### CODE REVIEW

**Comment by aubergine10** [comment]
Added v11 alphas. Not sure what we should do with main v11.0 release as it would be huge list so currently put "everything in the alphas" to keep it short.

On v11 STABLE workshop page I suggest just noting some of the main changes/features/fixes and referring users to github changelog if they want full details.

Will now try and finish original task as stated in OP...

**Comment by aubergine10** [comment]
Note to self: Make sure changelog entry for PR #535 also references #390 and contributions from @Craxy. (Probably already does, just need to double check next time I'm editing the changelog).

**Comment by aubergine10** [comment]
Just needs final commits adding to changelog (and date) for when we release 11.0

**Comment by aubergine10** [comment]
This now contains complete changelog ready for v11.0 release :)

Once merged we can publish v11 STABLE and v11 LABS

Text & images for v11 STABLE page are here: https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/issues/609

**Comment by aubergine10** [comment]
Ok, I still have some stuff to check today so tomorrow works good for me too.

Just spotted few more typos in changelog will update shortly

**Comment by krzychu124** [comment]
We should update version string too :)

**Comment by aubergine10** [comment]
Good catch! I've bumped version in this PR rather than start another one

**Comment by aubergine10** [comment]
Ah, need to rebase. Will also need to update game version constants in the `TrafficManagerMod.cs`.

File: TLM/TLM/TrafficManagerMod.cs [hunk 1]
```diff
@@ -9,6 +9,17 @@ namespace TrafficManager {
...
+        public static readonly uint GameVersionA = 1u;
+        public static readonly uint GameVersionB = 12u;
+        public static readonly uint GameVersionC = 1u;
+        public static readonly uint GameVersionBuild = 2u;
+
+        // Note: `Version` is also used in UI/MainMenu/VersionLabel.cs
+        public static readonly string Version = "11.0";
 
     public class TrafficManagerMod : IUserMod {
 #if LABS
```
Comment (krzychu124) [inline TLM/TLM/TrafficManagerMod.cs]: Here, duplicated code

**Comment by aubergine10** [comment]
Fixed up my mess :)

**Pull request merged**